x-pack/plugins/security_solution/public/common/components/endpoint/host_isolation/isolate_success.tsx

```diff
@@ -7,27 +7,44 @@
 
 import React, { memo, ReactNode } from 'react';
 import { EuiButtonEmpty, EuiCallOut, EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
-import { GET_SUCCESS_MESSAGE } from './translations';
+import { GET_ISOLATION_SUCCESS_MESSAGE, GET_UNISOLATION_SUCCESS_MESSAGE } from './translations';
 
 export interface EndpointIsolateSuccessProps {
   hostName: string;
+  isolateAction?: 'isolateHost' | 'unisolateHost';
   completeButtonLabel: string;
   onComplete: () => void;
   additionalInfo?: ReactNode;
 }
 
 export const EndpointIsolateSuccess = memo<EndpointIsolateSuccessProps>(
-  ({ hostName, onComplete, completeButtonLabel, additionalInfo }) => {
+  ({
+    hostName,
+    isolateAction = 'isolateHost',
+    onComplete,
+    completeButtonLabel,
+    additionalInfo,
+  }) => {
     return (
       <>
-        <EuiCallOut
-          iconType="check"
-          color="success"
-          title={GET_SUCCESS_MESSAGE(hostName)}
-          data-test-subj="hostIsolateSuccessMessage"
-        >
-          {additionalInfo}
-        </EuiCallOut>
+        {isolateAction === 'isolateHost' ? (
```

Comment:
Both of the returned components are exactly the same. Suggestion - do the conditional against the `title` value only and define only one component:

```jsx
return (
<>
  <EuiCallOut
    iconType="check"
    color="success"
    title={
      isolateAction === 'isolateHost' 
      ? GET_ISOLATION_SUCCESS_MESSAGE(hostName)
      : GET_UNISOLATION_SUCCESS_MESSAGE(hostName)
    }
    data-test-subj="hostIsolateSuccessMessage"
  >
    {additionalInfo}
  </EuiCallOut>

  //....
)
```